src/action/app-storage.js

```diff
@@ -44,6 +48,10 @@ class AppStorage {
     try {
       const state = JSON.stringify(this._store.settings);
       await this._AsyncStorage.setItem('settings', state);
+      await this._AsyncStorage.setItem(
+        'restoringWallet',
+        this._store.wallet.restoring
+      );
```

Comment:
We currently only persist the settings object and should keep it that way for consistency's sake. If additional attributes are added, they should be added on the settings object.

Reply:
Changed `restoring` to live in the `settings` part of the `store`.